portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -115,41 +110,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Redirect back to the HTML page.
     response.sendRedirect("/comments.html");
   }
-
-  /** Returns a URL that points to the uploaded file, or null if the user didn't upload a file. */
-  private String getUploadedFileUrl(HttpServletRequest request, String formInputElementName) {
+  
+  /**
+   * Returns the String representation of the blobKey of the file uploaded by the user to Blobstore. 
+   * Returns null if the user did not upload a file.
+   */
+  private String getUploadedBlobKey(HttpServletRequest request, String formInputElementName) {
     BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
     Map<String, List<BlobKey>> blobs = blobstoreService.getUploads(request);
     List<BlobKey> blobKeys = blobs.get(formInputElementName);
-
-    // User submitted form without selecting a file, so we can't get a URL. (dev server)
+    
+    // User submitted form without selecting a file, so we can't get a blobKey. (dev server)
     if (blobKeys == null || blobKeys.isEmpty()) {
       return null;
     }
 
-    // Our form only contains a single file input, so get the key at the first index.
+    // Our form only contains a single file input, so get the first index.
```

Comment:
I think you still want to say "get the key at the first index"

Reply:
right